scripts/compile-templates.mjs

```diff
@@ -0,0 +1,18 @@
+import fs from 'fs';
+import { exec }  from 'child_process';
+
+const sourceDir = (process.platform !== "win32")
+  ? "src/exporters"
+  : "src\\exporters";
```

Comment:
nit: you can use [`path.join`](https://nodejs.org/api/path.html#pathjoinpaths) to avoid having to do this detection yourself.

Reply:
Yes, but I would need a polyfill for path for the browser. So far I managed to get everything working with only a polyfill for `url`, but I should probably try to eliminate that one as well.